### PR TITLE
Added verbose logs instructions for helm installs

### PR DESCRIPTION
**Pull request by mmfred**
This PR addresses the request in Issue 3183

I added this section: http://localhost:8000/docs/integrations/kubernetes-integration/troubleshooting/get-logs-version/#verbose-helm

Question: Do we need steps like 5-9 from the previous section on verbose logs using a manifest file?

### CODE REVIEW

File: src/content/docs/integrations/kubernetes-integration/troubleshooting/get-logs-version.mdx [hunk 1]
```diff
@@ -48,7 +48,28 @@ To generate verbose logs and get version and configuration information, follow t
...
+    <Callout variant="caution">
+      Verbose mode significantly increases the amount of info sent to log files. Temporarily enable this mode only for troubleshooting purposes, and reset the log level when finished.
+    </Callout>
+  To get verbose logging details for an integration using Helm:
 
+    1. Enable verbose logging:
+    ```
+    helm upgrade -n <namespace> --reuse-values newrelic-bundle --set newrelic-infrastructure.verboseLog=true newrelic/nri-bundle
+    ```
+    2. Gather logs from the pod connecting to [Kubernetes state metrics](#logs-pod-kubestatemetrics) as described below.
```
Comment (roobre) [inline src/content/docs/integrations/kubernetes-integration/troubleshooting/get-logs-version.mdx:67]: Depending on the issue being debugged, logs for other pods might also be interesting.

Comment (greenszpila) [reply src/content/docs/integrations/kubernetes-integration/troubleshooting/get-logs-version.mdx:67]: I understand that we enable verbosity only for the infra pod and then we get logs from it. `kubectl logs newrelic-infra-xxxxx` Do we have to enable verbosity for the other pods too? The verbosity flag seems not available for the infra operator pod.

Comment (roobre) [reply src/content/docs/integrations/kubernetes-integration/troubleshooting/get-logs-version.mdx]: Enabling logs for the infra agent is usually enough. The key point is that the infra agent `DaemonSet` may comprise more than one `Pod`, especifically one per node that is in the cluster. Sometimes it is useful to have the logs for those other pods as well! :)

Comment (mmfred) [reply src/content/docs/integrations/kubernetes-integration/troubleshooting/get-logs-version.mdx]: I created this confusion by overly truncating the information I was attempting to follow, from the verbose logs using manifest file section. I've updated the steps. Please take a look and if it looks correct now, I'll merge.

Comment (mmfred) [reply src/content/docs/integrations/kubernetes-integration/troubleshooting/get-logs-version.mdx]: @roobre and @greenszpila - You can see the build here: https://pr-3252.d1vx4w5vgyla4s.amplifyapp.com/docs/integrations/kubernetes-integration/troubleshooting/get-logs-version/#verbose-helm

**Pull request merged**